packages/perseus/src/widgets/__tests__/grapher.cypress.jsx

```diff
@@ -27,7 +27,7 @@ const LINES = "[data-interactive-kind-for-testing=movable-line] > svg > path";
 describe("Grapher widget", () => {
     beforeEach(() => {
         Perseus.init({skipMathJax: true});
-        Perseus.Dependencies.setDependencies(webappDependencies);
+        Perseus.Dependencies.setDependencies(testDependencies);
```

Comment:
I suspect this might not work (pretty sure I had to use webappDependencies when I was getting Cypress working after we introduced dependencies). But we can fix this when we get to the Cypress setup ticket.

Reply:
If that's the case then we need to have a version of `testDependencies` that does more robust mocking.